src/java/picard/util/IntervalListTools.java

```diff
@@ -46,15 +46,13 @@
             "Both interval_list and VCF files are accepted as input. The interval_list file format is relatively simple" +
             " and reflects the SAM alignment format to a degree.  A SAM style header must be present in the file that " +
             "lists the sequence records against which the intervals are described.  After the header, the file then" +
-            " contains records, one per line in text format with the following" +
-            " values tab-separated: " +
-            "<pre>" +
-            "     -Sequence name (SN) <br />" +
-            "     -Start position (1-based)** <br />" +
-            "     -End position (1-based, end inclusive) <br />" +
-            "     -Strand (either + or -) <br />" +
-            "     -Interval name (ideally unique names for intervals)" +
-            "</pre>" +
+            " contains records, one per line in text format as shown below: " +
+
+            "<p><table> <tbody> <tr><th>Sequence name (SN)</th><th>Start position (1-based)</th><th>End position " +
+            "(1-based, end inclusive)</th><th>Strand (either + or -)</th><th>Interval name (an ideally unique name for the interval)</th></tr>" +
```

Comment:
I hesitate to let this pass before I see how this will look in the commandline. I know that there's a solution "coming", but right now this seems unreadable from the commandline.